pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -268,12 +268,12 @@ in
 
   clion = buildClion rec {
     name = "clion-${version}";
-    version = "2020.2.5"; /* updated by script */
+    version = "2020.3"; /* updated by script */
```

Comment:
looks like this will need to be changed:
```
in/sh" to "/nix/store/r3j288vpmczbl500w6zz89gyfa4nr0b1-bash-4.4-p23/bin/sh"
  checking for references to /build/ in /nix/store/lvn7rxms4ac3drm965684dvglw44bahi-clion-2020.3...
  patchelf: missing filename
  builder for '/nix/store/c9man4kz1wn24nlxa3ajsbjpqm7d7rpx-clion-2020.3.drv' failed with exit code 1
```
```
1 package failed to build:
jetbrains.clion

4 packages built:
jetbrains.goland jetbrains.phpstorm jetbrains.pycharm-community jetbrains.pycharm-professional
```

Reply:
Huh not sure how to solve this

Reply:
I'll take a look